### PR TITLE
fix(allcompany): company related data and users

### PR DESCRIPTION
**Pull request by okhun**

### CODE REVIEW

File: src/allCompanies/company/company_feedback.ts [hunk 1]
```diff
@@ -1,31 +1,32 @@
...
+  fullname: string
+  vehicle: VehiclePayload
+}
+export interface CountInterface {
+  excellent: number
+  good: number
+  normal: number
+  bad: number
+  worst: number
+}
```
[the remark this answers is not in the document]
Comment (okhun) [reply src/allCompanies/company/company_feedback.ts:18]: we did this kind of numbering, we changed to this, Shoxrux aka suggeted this format

[the remark this answers is not in the document]
Comment (okhun) [reply src/allCompanies/company/company_feedback.ts:18]: It should be name

File: src/allCompanies/company/company_feedback.ts [hunk 2]
```diff
@@ -1,31 +1,32 @@
-import { PageablesDto } from '~/shared/pageable.dto'
-import { UserPayload } from './company'
+export interface VehiclePayload {
+  mark: string
```
[the remark this answers is not in the document]
Comment (okhun) [reply src/allCompanies/company/company_feedback.ts:2]: I changed mark to make

[the remark this answers is not in the document]
Comment (okhun) [reply src/allCompanies/company/company_feedback.ts:2]: We do not need model_id

**Review by mrrashidov — APPROVED** [review]

**Pull request merged**